tests/jointcalTestBase.py

```diff
@@ -117,9 +117,14 @@ def _testJointCalTask(self, nCatalogs, dist_rms_relative, dist_rms_absolute, pa1
         oldWcsList = result.resultList[0].result.oldWcsList
 
         rms_result = self.jointcalStatistics.compute_rms(data_refs, self.reference)
+
+        # Make plots before testing, if requested, so we still get plots if tests fail.
+        if self.do_plot:
+            plot_dir = os.path.join('.test', self.__class__.__name__, 'plots')
+            if not os.path.isdir(plot_dir):
+                os.mkdir(plot_dir)
+            self.jointcalStatistics.make_plots(data_refs, oldWcsList, name=caller, outdir=plot_dir)
```

Comment:
Users have to explicitly turn on plotting (each test has a `plots=False` in its `setUp`), but being explicit about where those plots go is a good idea. Added a print statement.